broadinstitute/warp
# staging -> master

* update to new docker image

* Np get dev in sync with staging (#838)

* make dev and staging in sync

* version

* Np ultima cramonly test (#823)

* add validation and test wdl for cram only ug pipeline

* add testing wdl

* add to pipelineTestType

* add to changed workflows

* renaming

* update changed pipeline script syntax

* update changed pipeline script syntax

* add check to make sure bams are null and fail if they are not null

* Fix for large Ultima samples in Single Sample pipeline (#835)

* folding optional output bam into MarkDuplicates

* updating somatic test paths

* updating changelogs

* updating versions in wdl

* missed a few

* increase version for external whole genome reprocessing

Co-authored-by: jessicaway <jway@broadinstitute.org>

* add in load_tag fields

* update change logs for arrays and imputation internal

* update all other change logs

* update validate chip

* Force fragment size histogram file to exist in rnaseqc output (#840)

* force fragmentSizes histogram file to exist

* Adding filtering to Ultima Joint Genotyping pipeline (#804)

* Adding filtering to Ultima Joint Genotyping pipeline

Co-authored-by: jessicaway <jway@broadinstitute.org>

* Adding scientific test for CramOnly Ultima pipeline and adding it to dockstore (#844)

* GL-702: Update references to data stored in HCA buckets (#836)

* Update references to data stored in HCA buckets to copies made to broad-gotc-test-storage

* GL-1988: Update references to files in broad-gotc-test-storage/tmp/jbx (#848)

* Update references to files in broad-gotc-test-storage/tmp/jbx to new locations

* update docker images for internal arrays and internal imputation

* add lab_batch into internal arrays

* remove lab_batch

* Update Optimus Overview (#852)

Optimus Overview now matches dev version

* Lk ss2 fix broken link (#853)

Fixed broken link to python script.

* remove workspace bucket as input to ingestoutputstoTDR

* remove workspace bucket from wide table ingest

* recommit to refresh dockstore

* update versions for broad internal arrays and imputation and changelogs

* update changelogs

* Allow nested inputs in the imputation pipeline (#857)

* allow nested inputs in the imputation pipeline

* need to copy over pipeline text metrics to truth bucket (#856)

Co-authored-by: Sushma Chaluvadi <schaluva@broadinstitute.org>
Co-authored-by: meganshand <mshand@broadinstitute.org>
Co-authored-by: jessicaway <jway@broadinstitute.org>
Co-authored-by: Sushma Chaluvadi <43828038+schaluva@users.noreply.github.com>
Co-authored-by: kachulis <39926576+kachulis@users.noreply.github.com>
Co-authored-by: Tim Wang <31369366+timaeusx@users.noreply.github.com>
Co-authored-by: ekiernan <55763654+ekiernan@users.noreply.github.com>
